integreat_cms/static/src/css/style.scss

```diff
@@ -334,8 +334,6 @@ label:not([for]) {
     }
 
     table {
-        width: inherit !important;
```

Comment:
Note that this css rule does not apply to all tables. Instead, it only applies to tables which are children of a node with a table-listing class (see here: https://github.com/digitalfabrik/integreat-cms/blob/88a67cdac9e5e7313ba2b175712ee601106b9c0d/integreat_cms/static/src/css/style.scss#L321)

Of those tables, all have the `w-full` class.

Unfortunately, the change still caused an error, because it accidentally fixed another bug, which caused the poi table rendering to go wrong :sob: 
I have opened a pr to fix that at #3044

Reply:
oh, I forgot that this is ***S***CSS when I saw this :sweat_smile: